src/app/components/file-uploader/file-uploader.component.ts

```diff
@@ -0,0 +1,157 @@
+import { Component, Input, ViewChild, ElementRef, OnInit } from '@angular/core';
+import { UploadFile } from 'src/app/models/domain/uploadFile';
+import { FileUploaderService } from 'src/app/services/file-uploader.service';
+import { map } from 'rxjs/operators';
+import { HttpEventType } from '@angular/common/http';
+import { forkJoin, Observable, of } from 'rxjs';
+import { AlertConfig } from '../../models/internal/alert-config';
+import { AlertType } from '../../models/internal/alert-type';
+import { AlertService } from '../../services/alert.service';
+
+@Component({
+  selector: 'app-file-uploader',
+  templateUrl: './file-uploader.component.html',
+  styleUrls: ['./file-uploader.component.scss']
+})
+export class FileUploaderComponent {
+
+  @Input() acceptMultiple: boolean;
+  @Input() acceptedTypes: Array<String>;
+  @ViewChild('fileDropRef') fileInput: ElementRef;
+
+  maxFileSize = 5242880;
```

Comment:
No all properties/methods have their proper access modifiers. The order of the properties/methods should follow the guidelines defined here: https://github.com/DigitalExcellence/dex-frontend/wiki/Coding-Guidelines#-methods-and-variables-order